t/app/controller/report_as_other.t

```diff
@@ -155,6 +263,7 @@ sub start_report {
     FixMyStreet::override_config {
         ALLOWED_COBRANDS => [ 'fixmystreet' ],
         MAPIT_URL => 'http://mapit.uk/',
+        PHONE_COUNTRY => 'GB',
```

Comment:
Hmm, guess we should document that you'll have to set `PHONE_COUNTRY` if you want to use report as another user with phone numbers.